k8-helm-manifest/values.yaml

```diff
@@ -21,4 +21,5 @@ resources:
     cpu: "100m"
 mongodb:
   MONGODB_HOSTNAME: "mongodb"
-  MONGODB_DATABASE: "todo"
\ No newline at end of file
+  MONGODB_DATABASE: "todo"
+  MONGODB_ROOT_USERNAME: "root"
```

Comment:
Pls dont add it here.. You only need to add it in the deployment.yaml file

Reply:
ok